add-on/chrome/bootstrap.js

```diff
@@ -1039,6 +1039,14 @@ function loadDefaultPrefs() {
       }
     }
   });
+
+  if (Services.vc.compare(Services.appinfo.version, "47.0a1") < 0) {
+    branch.setBoolPref("loop.remote.autostart", false);
```

Comment:
I believe e10s is currently enabled for a few users on 46. I think it gets turned off around b6 time, though I'm not sure (b6 will be after we ship this).

As I'm not sure, I don't want us turning it on by accident and causing lots of crashes/other issues ;-)